perma_web/frontend/stores/globalStore.js

```diff
@@ -1,6 +1,17 @@
 import { reactive } from 'vue'
 
 export const globalStore = reactive({
+  captureStatus: 'ready',
+
+  updateCapture(state) {
+    this.captureStatus = state
+  },
+
+  captureErrorMessage: '',
+  updateCaptureErrorMessage(message) {
+    this.captureErrorMessage = message
+  },
```

Comment:
I think if this needs to be an array of errors, that should be easy to update in the future when I implement all possible Perma uploading/capture/archive related errors.